feed.rss

```diff
@@ -0,0 +1,179 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
The feed still is considered as no valid by the Google feed detection extension.

Reply:
Where are you checking this?

I use https://validator.w3.org/feed/check.cgi